File: Programmer_test/PT_Java/P12912.java
public class P12912 {
    public static void main(String[] args) {
        P12912 T = new P12912();
        System.out.println(T.solution(3, 5));
    }

    public long solution(int a, int b) {
        long answer = 0;
        int first = Math.min(a, b);
        int second = Math.max(a, b);
        for (int i = first; i < second + 1; i++) {
            answer += i;
        }
        return answer;
    }
}

File: Programmer_test/PT_Python/P12912.py
def solution(a, b):
    answer = 0
    fr = min(a,b)
    se = max(a,b)
    for i in range(fr, se+1):
        answer += i
    return answer

print(solution(3,5))
print(solution(3,3))
print(solution(5,3))